src/components/GlobalNavigation/components/MobileView/components/ArticleCard/ArticleCard.spec.tsx

```diff
@@ -1,8 +1,49 @@
-// import React from 'react';
-// import { render, screen } from '@testing-library/react';
-// import { ArticleCard } from './ArticleCard';
-// import { ArticleCardFixture } from './ArticleCard.fixture';
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { ArticleCard } from './ArticleCard';
+import { ArticleCardFixture } from './ArticleCard.fixture';
+import userEvent from '@testing-library/user-event';
 
-describe('<GlobalNavigation.MobileView.ArticleCard />', () => {
-  it.todo('should assert something');
+describe('<ArticleCard />', () => {
+  const TestBed = ({ isVisible }: { isVisible?: boolean }) => (
+    <ArticleCard
+      id={ArticleCardFixture.id}
+      isVisible={isVisible}
+      label={ArticleCardFixture.label}
+      metaLabel={ArticleCardFixture.metaLabel}
+      title={ArticleCardFixture.title}
+      url={ArticleCardFixture.url}
+    />
+  );
+  it('should render base component correctly', () => {
+    const { container } = render(<TestBed />);
+
+    expect(container).not.toBeEmptyDOMElement();
+  });
+
+  it('should navigate to the article card by tab clicking', () => {
+    render(
+      <>
+        <TestBed />
+      </>,
+    );
```

Comment:
I don't think you need to wrap it in a fragment, right?
```suggestion
    render(<TestBed />);
```